SharedClusterSnapshotRestoreIT » testParallelRestoreOperationsFromSingleSnapshot  fails on windows

```
java.util.concurrent.ExecutionException: RemoteTransportException[[node_s1][127.0.0.1:50671][cluster:admin/snapshot/restore]]; nested: RepositoryException[[test-restore-snapshot-repo] could not read repository data from index blob]; nested: AccessDeniedException[C:\Users\jenkins\workspace\elastic+elasticsearch+7.2+multijob-windows-compatibility\os\windows-2019\server\build\testrun\integTest\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_1D101C95ED2AB85B-001\tempDir-002\repos\BlgxpMTata\pending-incompatible-snapshots-WmL0FE7FTIG7hJpeydq34g -> C:\Users\jenkins\workspace\elastic+elasticsearch+7.2+multijob-windows-compatibility\os\windows-2019\server\build\testrun\integTest\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_1D101C95ED2AB85B-001\tempDir-002\repos\BlgxpMTata\incompatible-snapshots];Open stacktrace
Caused by: org.elasticsearch.transport.RemoteTransportException: [node_s1][127.0.0.1:50671][cluster:admin/snapshot/restore]Open stacktrace
Caused by: org.elasticsearch.repositories.RepositoryException: [test-restore-snapshot-repo] could not read repository data from index blobOpen stacktrace
Caused by: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\elastic+elasticsearch+7.2+multijob-windows-compatibility\os\windows-2019\server\build\testrun\integTest\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_1D101C95ED2AB85B-001\tempDir-002\repos\BlgxpMTata\pending-incompatible-snapshots-WmL0FE7FTIG7hJpeydq34g -> C:\Users\jenkins\workspace\elastic+elasticsearch+7.2+multijob-windows-compatibility\os\windows-2019\server\build\testrun\integTest\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_1D101C95ED2AB85B-001\tempDir-002\repos\BlgxpMTata\incompatible-snapshotsOpen stacktrace
```

https://scans.gradle.com/s/7ocgmvdtaatmc/tests/kyv2y2z3r4v7m-5qrcvit3in7tm
https://scans.gradle.com/s/rhypi43cmywgq/tests/kyv2y2z3r4v7m-5qrcvit3in7tm

--- a/server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java
+++ b/server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java
@@ -19,7 +19,6 @@
 
 package org.elasticsearch.snapshots;
 
-import org.apache.lucene.util.Constants;
 import org.apache.lucene.util.SetOnce;
 import org.elasticsearch.ElasticsearchException;
 import org.elasticsearch.ExceptionsHelper;
@@ -3800,7 +3799,6 @@ public class SharedClusterSnapshotRestoreIT extends AbstractSnapshotIntegTestCas
     }
 
     public void testParallelRestoreOperationsFromSingleSnapshot() throws Exception {
-        assumeFalse("https://github.com/elastic/elasticsearch/issues/44671", Constants.WINDOWS);
         String indexName1 = "testindex1";
         String indexName2 = "testindex2";
         String repoName = "test-restore-snapshot-repo";

